src/ui/CategoryFacet/CategoryValueRoot.ts

```diff
@@ -0,0 +1,21 @@
+import { CategoryValueParent, CategoryValue } from './CategoryValue';
+import { CategoryFacetTemplates } from './CategoryFacetTemplates';
+import { Dom } from '../../utils/Dom';
+import { CategoryChildrenValueRenderer } from './CategoryValueChildrenRenderer';
+
+export class CategoryValueRoot implements CategoryValueParent {
+  private children: CategoryValue[] = [];
+  public categoryChildrenValueRenderer: CategoryChildrenValueRenderer;
+
+  constructor(element: Dom, categoryFacetTemplates: CategoryFacetTemplates) {
+    this.categoryChildrenValueRenderer = new CategoryChildrenValueRenderer(element, categoryFacetTemplates, this);
+  }
+
+  public getPath(partialPath: string[] = []) {
+    return partialPath;
```

Comment:
?

Reply:
Suppose you have this: CategoryValueRoot <- CategoryValue1 <- CategoryValue2.

`getPath` on CategoryValue2 will call `CategoryValue1.getPath([CategoryValue2.value)]`

CategoryValue1 will then call 
`CategoryValueRoot.getPath([CategoryValue1.value, CategoryValue2.value])`

And then CategoryValueRoot will simply return the path it was called with since it does not hold any value.